docs/Datadog.Trace/README.md

```diff
@@ -0,0 +1,220 @@
+
+Update your code to the following:
+
+```csharp
+using Datadog.Trace;
+
+// Create your settings as before
+var settings = new TracerSettings();
+
+// var tracer = new Tracer(settings) // <- Delete this line
+Tracer.Configure(settings);          // <- Add this line
+```
+
+### TracerSettings are immutable
+
+In version 1.x the `TracerSettings` object passed to a `Tracer` instance could be subsequently modified. Depending on the changes, these may or may not have been respected. In version 2.0, an `ImmutableTracerSettings` object is created when the `Tracer` instance is configured. Subsequent changes to the `TracerSettings` instance will not be observed by `Tracer`. 
+
+This change should not generally require changes to your code, but the type of `Tracer.Settings` is now an `ImmutableTracerSettings` instance, not a `TracerSettings` instance.
+
+```csharp
+using Datadog.Trace;
+
+var settings = TracerSettings.FromDefaultSources();
+settings.TraceEnabled = false;   // TracerSettings are mutable 
+
+Tracer.Configure(settings);
+
+// All properties on Settings are now read-only 
+// Tracer.Instance.Settings.TraceEnabled = false; // <- DOES NOT COMPILE
+```
+
+### ADO.NET integrations can be disabled individually
+
+In version 1.x, you could disable automatic instrumentation of all ADO.NET integrations using the `AdoNet` integration ID. From version 2.0, this integration ID has been removed and replaced with the following integration IDs:
+
+* `MySql`
+* `Npgsql` (PostreSQL)
+* `Oracle`
+* `SqlClient` (SQL Server)
+* `Sqlite`
+
+This enables you to disable specific ADO.NET integrations if required. [See our documentation for a complete list of supported integration IDS](https://docs.datadoghq.com/tracing/setup_overview/compatibility_requirements/dotnet-core/#integrations).
+
+This change also removes the now-obsolete `TracerSettings.AdoNetExcludedTypes` setting, and corresponding environment variable `DD_TRACE_ADONET_EXCLUDED_TYPES`. Replace usages of these (for an example integration ID `MyIntegrationId`) with `TracerSettings.Integrations["MyIntegrationId"].Enabled` and `DD_TRACE_MyIntegrationId_ENABLED` respectively: 
+
+```csharp
+using Datadog.Trace;
+
+var settings = TracerSettings.FromDefaultSources();
+
+settings.AdoNetExcludedTypes.Add("MySql");      // <- Delete this line
+settings.Integrations["MySql"].Enabled = false; // <- Add this line
+```
+
+### ElasticsearchNet5 integration ID has been removed
+
+In version 1.x, the integration ID for `Elasticsearch.Net` in version 5.x was `ElasticsearchNet5`, and for version 6.x+ was `ElasticsearchNet`. From version 2.0, support for the ID `ElasticsearchNet5` has been removed, and instead `ElasticsearchNet` now can be used to  
```

Comment:
can be used to...?

Reply:
Has been resolved by Andrew in one commit